app/PAWAZI/PAWAZI_task_2.c

```diff
@@ -0,0 +1,78 @@
+/**
+ * Responsible: Pawel Aziukiewicz
+ *
+ *
+ * History log:
+ *  06.12.2017 initial version
+ */
+
+#include <stdio.h>
+#include "PAWAZI_task_2.h"
+
+
+eErr_t PAWAZI_setLampBit(eLamp_t bit, int* reg){
+
+	eErr_t res = E_OK;
+	
+	if(bit<LAMP_START || bit>=LAMP_COUNT){
+		res = E_INVALID_ARG_1;
+	}else if(reg == NULL){
+		res = E_INVALID_ARG_2;
+	}else {
+		*reg |= 1<<bit;
+	}
+	
+	return res;
+	
+	
+}
+
+eErr_t PAWAZI_clearLampBit(eLamp_t bit, int* reg){
```

Comment:
Accepted but next time please use hungarian notationalso for function arguments names